bin/build.js

```diff
@@ -18,6 +18,12 @@ if ( ! babelLoader ) {
 	throw new Error( 'Couldn\'t find the babel loader config.' );
 }
 
+console.log( 'Adding transpilable dependencies to babel include path' );
+babelLoader.include = [
+	babelLoader.include,
```

Comment:
good catch! but in this case no because the value in the imported config from `create-react-app` uses a string for the `include` field, so we'd be spreading that string into its constituent characters.

this config will need to change after we get things working again because `react-scripts` diverged dramatically since v0.9, but for now I just want to kind of embrace it and fix it, then when starting from a working spot, get back to the build system.